src/build_options.rs

```diff
@@ -631,26 +655,13 @@ pub fn find_interpreter(
 ) -> Result<Vec<PythonInterpreter>> {
     match bridge {
         BridgeModel::Bindings(binding_name, _) => {
-            let mut interpreter = if !interpreter.is_empty() {
-                PythonInterpreter::check_executables(interpreter, target, bridge)
-                    .context("The given list of python interpreters is invalid")?
-            } else {
-                PythonInterpreter::find_all(target, bridge, min_python_minor)
-                    .context("Finding python interpreters failed")?
-            };
-
-            if interpreter.is_empty() {
-                if let Some(minor) = min_python_minor {
-                    bail!("Couldn't find any python interpreters with version >= 3.{}. Please specify at least one with -i", minor);
-                } else {
-                    bail!("Couldn't find any python interpreters. Please specify at least one with -i");
-                }
-            }
-
+            let mut interpreters = Vec::new();
             if binding_name.starts_with("pyo3") && target.is_unix() && target.cross_compiling() {
                 if let Some(cross_lib_dir) = std::env::var_os("PYO3_CROSS_LIB_DIR") {
```

Comment:
Should there also be a check for `PYO3_CONFIG_FILE` env var, and just skip all this if so?

Reply:
I think that'd be nice, but it's out of scope of this PR, I'll open a new issue to track it.